cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -514,6 +515,144 @@
+      "File \u001b[0;32m~/opt/anaconda3/envs/lattice/lib/python3.9/site-packages/IPython/core/interactiveshell.py:284\u001b[0m, in \u001b[0;36m_modified_open\u001b[0;34m(file, *args, **kwargs)\u001b[0m\n\u001b[1;32m    277\u001b[0m \u001b[39mif\u001b[39;00m file \u001b[39min\u001b[39;00m {\u001b[39m0\u001b[39m, \u001b[39m1\u001b[39m, \u001b[39m2\u001b[39m}:\n\u001b[1;32m    278\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\n\u001b[1;32m    279\u001b[0m         \u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mIPython won\u001b[39m\u001b[39m'\u001b[39m\u001b[39mt let you open fd=\u001b[39m\u001b[39m{\u001b[39;00mfile\u001b[39m}\u001b[39;00m\u001b[39m by default \u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m    280\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39mas it is likely to crash IPython. If you know what you are doing, \u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m    281\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39myou can use builtins\u001b[39m\u001b[39m'\u001b[39m\u001b[39m open.\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m    282\u001b[0m     )\n\u001b[0;32m--> 284\u001b[0m \u001b[39mreturn\u001b[39;00m io_open(file, \u001b[39m*\u001b[39;49margs, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n",
+      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'ref_files/3M-february-2018.txt'"
+     ]
+    }
+   ],
+   "source": [
+    "#Code in sample notebook consumes the barcodes across all 3 lists and compiles them into a table\n",
+    "ref_dir = 'ref_files/'\n",
+    "v2_file = '737K-august-2016.txt'\n",
+    "v3_file = '3M-february-2018.txt'\n",
+    "multiome_file = '737K-arc-v1.txt'\n",
+    "\n",
+    "v2_list = [line.strip() for line in open(ref_dir + v2_file, 'r')]\n",
+    "v3_list = [line.strip() for line in open(ref_dir + v3_file, 'r')]\n",
+    "multiome_list = [line.strip() for line in open(ref_dir + multiome_file, 'r')]\n",
+    "\n",
+    "#create barcode results dictionary\n",
+    "barcode_results = {\n",
+    "    'barcode':[],\n",
+    "    '3pv2_5pv1_5pv2':[],\n",
+    "    '3pv3':[],\n",
+    "    'multiome':[]\n",
+    "}\n",
+    "\n",
+    "#extract matrix barcodes\n",
+    "barcodes = adata.obs.index.to_series()  # this code assumes the obs index is of the n_obs barcodes\n",
```

Comment:
I wouldn't expect this task to consume any `adata` because it should just be creating a table from the v2/v3/multiome files